### ISSUE TITLE
Can't get Ip from router/Doesn't connect to router

### ISSUE THREAD

**Issue by realchandan** [raspberrypi/rpi-imager#859]
### What happened?

I'm tired of flashing my SD card over and over again. I've flashed them dozens of times, and it can't just be a coincidence. I have both a Raspberry Pi 5 and a Raspberry Pi 4.

Problem: When I flash my SanDisk 64 GB memory card with the image file [2024-03-15-raspios-bookworm-arm64-lite.img](https://downloads.raspberrypi.com/raspios_lite_arm64/images/raspios_lite_arm64-2024-03-15/2024-03-15-raspios-bookworm-arm64-lite.img.xz), using custom options (see screenshots), I can't seem to get my Raspberry Pi to obtain an IP address from the router, either through WiFi or LAN. It doesn't even show up in the router. I've rebooted my router as well, but the problem persists. My router is running OpenWrt.

![image](https://github.com/raspberrypi/rpi-imager/assets/93002151/2fad3604-fcb4-429e-99e6-3b734054d141)
![image](https://github.com/raspberrypi/rpi-imager/assets/93002151/2667cb0e-78e8-4f24-8082-eef80d256f17)

### Version

1.8.5 (Default)

### What host operating system were you using?

Windows

### Host OS Version

Windows 10 Home Version 22H2

### Selected OS

Raspberry Pi OS Lite 64-bit

### Which Raspberry Pi Device are you using?

Raspberry Pi 5

### What kind of storage device are you using?

microSD Card in a USB reader

### OS Customisation

- [ ] Yes, I was using OS Customisation when the bug occurred.

**Comment by lurch** [comment]
I wonder if this might be a problem with Raspberry Pi OS (or your router), rather than a problem with Raspberry Pi Imager?
Is your Raspberry Pi able obtain an IP address from your router over LAN if you **don't** set any customisation options in Raspberry Pi Imager?

**Comment by lurch** [comment]
P.S. If you're able to (temporarily) attach an HDMI monitor/TV and a USB keyboard to your Raspberry Pi, that'll make it _much_ easier to diagnose what's going on.

**Comment by lurch** [comment]
> I've been dealing with this issue for months now

Have you tried asking for help on the [forums](https://forums.raspberrypi.com/) ?

**Comment by XECDesign** [comment]
What channel/frequency is your wifi on?

I’m guessing it’s not one that is allowed in the region according to the regulatory domain blob.

Not sure why Ethernet wouldn’t work though. A lot more information would need to be provided to look into it. Maybe the output from journalctl would tell us something.

**Comment by XECDesign** [comment]
Tried to reproduce the issue here and everything just worked as expected. I used the settings you gave in the first screenshot and the same image running on a pi5. WLAN connected as expected and I was able to ssh in.

It would probably be a good idea to drill down and figure out why it's not working out of the box in your case, but if you're happy with your workaround and want to leave it here, no worries.
